modules/react/status-indicator/lib/StatusIndicator.tsx

```diff
@@ -21,10 +20,10 @@ export enum StatusIndicatorEmphasis {
 }
 
 export interface StatusIndicatorGenericStyle extends GenericStyle {
-  styles: CSSObject & React.CSSProperties;
+  styles: CSSProperties;
   variants: {
     [statusType in StatusIndicatorType]: {
-      [statusEmphasis in StatusIndicatorEmphasis]?: CSSObject & React.CSSProperties;
+      [statusEmphasis in StatusIndicatorEmphasis]?: CSSProperties;
```

Comment:
Sorry, I'm not following. Are these changes related to the hook?

Reply:
I was looking at what CSSProperties was, for use in my example. Looks like this file missed the conversion whenever y'all created it. Happy to pull this out if you think it is too unrelated.